tests/test_cache.py

```diff
@@ -51,14 +67,15 @@ def test_load_from_cache_with_path_not_complete_miss(self):
         with create_test_cache():
             cache_path = get_cached_path(TEST_MODEL_HANDLE)
 
-            # Should be a cache `miss` if the directory exists but not the marker file.
+            # Should be a cache `miss` if the directory and file exist but not the marker file.
             os.makedirs(cache_path)
             Path(os.path.join(cache_path, TEST_FILEPATH)).touch()  # Create file
 
             self.assertEqual(None, load_from_cache(TEST_MODEL_HANDLE, path=TEST_FILEPATH))
 
     def test_load_from_cache_with_complete_marker_no_files_miss(self):
         with create_test_cache():
+            # Why is this line needed?
```

Comment:
Was curious about this too! Will remove this comment.

Reply:
You can remove it and same for the line #88 below. Another 🤦. Thanks for cleaning this up :)